src/mumble/Settings.cpp

```diff
@@ -1005,6 +1005,7 @@ void Settings::legacyLoad(const QString &path) {
 	// TalkingUI
 	LOAD(qpTalkingUI_Position, "ui/talkingUIPosition");
 	LOAD(bShowTalkingUI, "ui/showTalkingUI");
+	LOAD(bTalkingUI_UsersAlwaysVisible, "ui/talkingUI_UsersAlwaysVisible");
```

Comment:
You can remove this line as this is the loading routine for the legacy settings (in INI format). Mumble has switched to JSON and since your new option by definition not be included in legacy config files, you do not have to load it here.

Reply:
Oh alright, I was wondering why it was loaded at two locations, but blessed are the people who can read "legacyLoad". Perhaps I should actually use an IDE for mumble^^